rossum_api/models/queue.py

```diff
@@ -9,7 +9,7 @@ class Queue:
     id: int
     name: str
     url: str
-    workspace: str
+    workspace: Optional[str]
```

Comment:
Can we really have a queue without a workspace? :thinking:

Reply:
Sorry, I read the description and the docs in too much haste. Seems legit. merging.

Reply:
Yes, e.g. in our EXE testing org there's quite a lot of them.